index.js

```diff
@@ -75,17 +127,23 @@ var formatAnswer = function(winner, mode, placeDesc) {
     winnerEstimate = minutes.toString() + ' minute';
     winnerEstimate += minutes > 1 ? 's' : '';
   } else {
+    winner.estimate = (staging) ? winner.estimate * 2 : winner.estimate;
     var dollars = Math.floor(winner.estimate / 100);
-    var cents = winner.estimate % 100;
+    var cents = Math.floor(winner.estimate % 100);
```

Comment:
@cpruijsen yes, i added it because uber estimates occasionally returned fractional cents, which was really annoying haha